css/fonts.css

```diff
@@ -0,0 +1,11 @@
+@font-face {
+    font-family: 'Roboto Bold';
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/bugs-finde/pullRequest?prid=5808908)